src/utils.ts

```diff
@@ -98,6 +98,18 @@ export function stringifyComponentImport({ name, path, importName, sideEffects }
 }
 
 export function resolveOptions(options: Options, viteConfig: ResolvedConfig): ResolvedOptions {
+  if (options.transformer === 'svelte313') {
+    if (options.extensions) {
+      if (typeof options.extensions === 'string') {
+        // convert to array
+        options.extensions = [options.extensions]
+      }
+      options.extensions.push('svelte')
+    }
+    else {
+      options.extensions = 'svelte'
+    }
+  }
```

Comment:
Maybe we can change default options here instant of user options, that you don't need to resolve extensions twice.

Reply:
Sorry I don't get what you mean.

Reply:
Do you mean changing the `options.extensions` type to only `string[]`?

Reply:
Or do you mean simplifying all the highlighted code block to simply:
```typescript
  if (options.transformer === 'svelte313') {
      options.extensions = 'svelte'
  }
```
losing all other extensions.

Reply:
defaultOptions is a preset config for Vue, maybe we can have a preset for Svelte.

Reply:
so the default `options.extensions` would be `["vue", "svelte"]` instead of only `["vue"]`?

Reply:
That works too, or switch to svelte when is svelte transformer.

Reply:
> defaultOptions is a preset config for Vue, maybe we can have a preset for Svelte.

Oh I get what you mean. I'll try that.